docs/src/pages/components/Skeleton/SwiftUISkeletonPage.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Skeleton
+platform: ios
+documentationId: BPKSkeleton
+githubPath: Skeleton
+---
+
+
+import IOSComponentScreenshots from 'components/Screenshots/IOSComponentScreenshots';
+
+import Readme from 'backpack-ios/Backpack-SwiftUI/Skeleton/README.md';
+import screenshotDefault from 'backpack-ios/screenshots/iPhone 8-swiftui_skeleton___default_lm.png';
+import screenshotDefaultDm from 'backpack-ios/screenshots/iPhone 8-swiftui_skeleton___default_dm.png';
+
+## Table of contents
+
+## Default
+
+<IOSComponentScreenshots
+  lightMode={screenshotDefault}
+  darkMode={screenshotDefaultDm}
+  altText="Default Skeleton" />
+
+
```

Comment:
Should there be a title here, or is that included in the readme?

Reply:
Yes, README has the title then.
![image](https://user-images.githubusercontent.com/13347837/190550858-341188a9-2977-497e-8fdc-a80e6eb72a6c.png)